cluster-diagnosis/sysdumpcollector.py

```diff
@@ -200,6 +202,27 @@ def collect_cilium_configmap(self):
             log.info("collected cilium configmap yaml file: {}".format(
                 configmap_file_name))
 
+    def collect_cilium_secret(self):
+        secret_file_name = "cilium-etcd-secrets-{}.json".format(
+            utils.get_current_time())
+        cmd = "kubectl get secret cilium-etcd-secrets -n kube-system -ojson"
+        try:
+            output = json.loads(subprocess.check_output(cmd, shell=True))
+            data = {}
+            for key, value in output.get('data').iteritems():
+                data[key] = "XXXXX"
+            output['data'] = data
+            with open(
+                os.path.join(self.sysdump_dir_name, secret_file_name), 'w') as fp:
+                fp.write(json.dumps(output))
+        except subprocess.CalledProcessError as exc:
+            if exc.returncode != 0:
+                log.error("Error: {}. Unable to get cilium secret json")
```

Comment:
print the error. `Error: {}` param for {} is missing.

Reply:
Change the log to "Unable to retrieve and redact cilium secret."

Reply:
Done